backend/src/main/java/com/bakdata/conquery/models/events/parser/specific/IntegerColumnValues.java

```diff
@@ -0,0 +1,27 @@
+package com.bakdata.conquery.models.events.parser.specific;
+
+import com.bakdata.conquery.models.events.parser.ColumnValues;
+import it.unimi.dsi.fastutil.ints.IntBigArrayBigList;
+
+class IntegerColumnValues extends ColumnValues<Integer> {
+	private final IntBigArrayBigList values = new IntBigArrayBigList();
+
+	protected IntegerColumnValues() {
+		super(0);
+	}
+
+	@Override
+	public Integer read(int event) {
+		return values.getInt(event);
+	}
+
+	@Override
+	protected void write(Integer obj) {
+		values.add(obj.intValue());
+	}
+
+	@Override
+	protected int countValues() {
+		return (int) values.size64();
+	}
```

Comment:
Sollte dann nciht eher ein long zurückgegeben werden?

Reply:
Ich wechsle es glaube ich wieder zu IntList, sehe aktuell keinen mehrwert in den LongLists